enhancements/two-node-fencing/tnf.md

```diff
@@ -565,6 +565,14 @@ possible, but it is not sufficient for the operation of a healthy cluster longte
 
 Because of the flexibility it can offer users to adjust their cluster capacity according to their needs, transitioning between control-plane topologies may be introduced for evaluation in a future enhancement.
 
+#### Regarding Compute Nodes
```

Comment:
Nit: I think it might be worth to clarify that this discussion is about `adding` additional compute nodes, which might be trivial but isn't stated

Reply:
I changed the title, but really it covers installing with compute nodes and adding them later.